### PR TITLE
Switch to using markdown

### LINKED ISSUE #4: Switch to markdown renderer
I'd certainly prefer dealing with markdown rather than HTML, and it should be possible: https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/setting-a-markdown-processor-for-your-github-pages-site-using-jekyll

### PR DESCRIPTION
**Pull request by gregcorbett**
Resolves #4 

This PR reworks the main page for GOCDB.github.io to use markdown instead of HTML, giving it a bit of an update at the same time.

It's viewable here: https://gregcorbett.github.io/GOCDB.github.io/

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Grid Operations Configuration Management Database.
+A Repository, Portal and REST style API for managing Grid and Cloud topology objects including; projects, administrative domains, sites, services, service-endpoints, service-groups, downtimes, users, roles and business rules.
+
+<span>
+<img alt="STFC logo" src="img/UKRI_STF_Council-Logo_Horiz-RGB_crop.png" height=57/>
```
Comment (ineilson) [inline index.md:5]:
```suggestion
<img alt="STFC logo" src="img/UKRI_STF_Council-Logo_Horiz-RGB_crop.png" height=64/>
```
Looks better IMHO.

**Review by ineilson — APPROVED** [review]

File: index.md (hunk 1 above)
Comment (tofu-rocketry) [reply index.md:5]: Yeah, I'd agree on that.

**Review by ineilson — APPROVED** [review]

**Comment by gregcorbett** [comment]
Or indeed, replace the travis build with GitHub actions 😄

**Comment by gregcorbett** [comment]
The GitHub actions were successful: https://github.com/gregcorbett/GOCDB.github.io/actions/runs/2676140839, but I guess won't show up on a PR untill this PR is merged,

**Review by tofu-rocketry — APPROVED** [review]

**Pull request merged**